.husky/post-merge

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-rm -rf node_modules && yarn install --frozen-lockfile
```

Comment:
Not sure why the article I used as reference is removing the folder in this hook but not in other hooks. I'm trying without removing it and will see.